talent/engineering-teams-descr.md

```diff
@@ -0,0 +1,38 @@
+
+### Data Retrieval Team 
+
+- The Data Retrieval team is developing source{d}'s high-level Engine for running scalable data retrieval pipelines that process and manipulate any number of code repositories for source code analysis. Written mostly in Scala, it aims to be robust, friendly and flexible: it is built on top of Apache Spark, accessible both via Scala and Python Spark APIs, and capable of running on large-scale distributed clusters over petabytes of data. 
+
+- This core tool will be used for building source{d}'s unique global scale open dataset of +60M code repositories for our Machine Learning research on source code. 
+
+### Language Analysis Team
+
+- The Language Analysis team is developing source{d} tools and libraries for generating and processing universal abstract syntax trees for programming languages. These components are integrated into the Babelfish Open Source project, written in Go, C++, Scala, and any other language that we're parsing.
+
+- Babelfish powers the cutting-edge research that our Machine Learning team does on their ambitious mission to provide Machine Learning on Source Code. Our tools also are being used by a community of developers, researchers, and companies across the world in order to do vanguard research and build the next generation of developer tools.
+
+### Infrastructure Team
+
+- The Infrastructure team focuses on managing and growing our bare metal computation cluster. This cluster allows us to run large scale computational algorithms and store the required information for those computations.
+
+- Our cluster, with close to 2000 threads, more than 11TB of RAM and 1PB of storage, is backed by CoreOS and Kubernetes. It has two main goals: on the one hand, it stores all the available public code in a HDFS cluster and on the other hand, it runs intensive computation jobs over the stored data on top of Spark.
+
+- Besides the main points above, the infrastructure team is also in charge of several other important services, such as databases, queues, CI, monitorization, logging, etc.
+
+### Communication 
+
+- The ```Applications``` team is responsible for building scalable developer tools on top of the research the ```Machine Learning``` team is doing. The ```Data Retrieval``` team retrieves and processes public source code in order to build the data pipelines for the Machine Learning team’s models. For that, BabelFish from ```Language Analysis``` team is used in order to generate the datasets as Universal Abstract Syntax tree to be analyzed directly or given as input data to train the ML models. All the necessary services and computational clusters for every team, are supported by the Infrastructure team.
```

Comment:
Imo this paragraph looks weird. Some parts are already mentioned in each team description (ie Apps team); in other cases, there is new data here that would be better in the team description (ie, DR team, retrieving and processing all the OS over the world)

Imo the relation between teams can be understood with the team descriptions + attached graph, so it would not be necessary an extra (redundant) paragraph.

Reply:
The idea of a graph sounds great. I would still keep the text though along with the graph I think. @dpordomingo do you use any specific tools to make graphs for documents here or I can use whatever is convenient for me? Let me know if you have any suggestions.

Reply:
....after discussing with @dpordomingo on slack, maybe @bzz you have any graph/drawing tools suggest so we can be consistent across the repos?